Header header
Detection2D[] detections    # List of 2D detections